### PR TITLE
feat: brake commands are now in percentages

### PR DESCRIPTION
**Pull request by jfjensen**

### CODE REVIEW

File: ros/src/twist_controller/dbw_node.py [hunk 1]
```diff
@@ -54,7 +54,7 @@ def __init__(self):
         vc.vehicle_mass = rospy.get_param('~vehicle_mass', 1736.35)
         vc.fuel_capacity = rospy.get_param('~fuel_capacity', 13.5)
         vc.brake_deadband = rospy.get_param('~brake_deadband', .1)
-        vc.decel_limit = rospy.get_param('~decel_limit', -5)
+        vc.decel_limit = rospy.get_param('~decel_limit', -1.)
```
Comment (eljefec) [inline ros/src/twist_controller/dbw_node.py:57]: Can you update dbw_sim.launch? The value there overrides the default specified in this line. Also, there was a snippet of John's team's config on #p-system-integration and they use accel/decel limits of 0.8 and -0.8.

**Review by eljefec — CHANGES_REQUESTED** [review]

File: ros/src/twist_controller/dbw_node.py (hunk 1 above)
Comment (jfjensen) [reply ros/src/twist_controller/dbw_node.py:57]: ok I'll look into it

File: ros/src/twist_controller/launch/dbw_sim.launch [hunk 2]
```diff
@@ -4,8 +4,8 @@
         <param name="vehicle_mass" value="1080." />
         <param name="fuel_capacity" value="0." />
         <param name="brake_deadband" value=".2" />
-        <param name="decel_limit" value="-5." />
-        <param name="accel_limit" value="1." />
+        <param name="decel_limit" value="-0.8" />
```
Comment (eljefec) [inline ros/src/twist_controller/launch/dbw_sim.launch]: Can you update dbw.launch so that Carla gets this value too?

Comment (eljefec) [inline ros/src/twist_controller/launch/dbw_sim.launch:7]: Can you update dbw.launch so that Carla gets this value too?

**Review by eljefec — CHANGES_REQUESTED** [review]

**Review by eljefec — APPROVED** [review]

**Pull request merged**